django-stubs/forms/models.pyi

```diff
@@ -150,7 +150,7 @@ class BaseModelFormSet(Generic[_M, _ModelFormT], BaseFormSet[_ModelFormT]):
     def save_existing(self, form: _ModelFormT, instance: _M, commit: bool = ...) -> _M: ...
     def delete_existing(self, obj: _M, commit: bool = ...) -> None: ...
     saved_forms: List[_ModelFormT] = ...
-    save_m2m: Callable[[self], None] = ...
+    def save_m2m(self) -> None: ...
```

Comment:
Context: I think that confusion came from these lines: https://github.com/django/django/blob/8c0886b068ba4e224dd78104b93c9638b860b398/django/forms/models.py#L781-L785

But, this is indeed a instance-level method. So, `self` is there.